docs/edge-compute/edge-agent-usage.md

```diff
@@ -0,0 +1,271 @@
+description: Learn more about running and using the Losant Edge Agent.
+
+# Losant Edge Agent Usage
+
+The Losant Edge Agent ("Agent") is a command line utility exposed through [Docker](https://www.docker.com/what-docker) as a [container](https://www.docker.com/what-container) you can run on your [Edge Compute Device](/devices/edge-compute/). The below outlines some example usage for how you can interact with the Agent on your device. For full documentation of all available flags and options, see the [`losant/edge-agent` image on Docker Hub](https://hub.docker.com/r/losant/edge-agent/).
+
+For help getting the Agent setup on your device, see the [installation instructions](/edge-compute/edge-agent-installation/). All command examples below assume you can run `docker` without `sudo` or that you are logged in as `root` (not recommended). If you wish to run `docker` as your current user, see [these instructions](/edge-compute/edge-agent-installation/#optional-executing-the-docker-command-without-sudo).
+
+## Creating Storage Area
+
+It is **strongly** recommended that you mount a volume in to the Agent as a workspace for persistent data. By default, the Agent will write data **inside** the container to `/data`. By [mounting a volume or using a volume container](https://docs.docker.com/storage/volumes/), we can have the Agent write to a local folder on the Edge Compute Device. An important benefit to this being that you have access to the persistent data created by the Agent and thus, containers can be destroyed and created without losing the data created at run time. Another benefit is that the Agent doesn't accumulate data internally which could impact the performance and/or stability of the container.
+
+All we need to do is create a folder locally to house the data. This folder can be anywhere on your filesystem, so long as permissions are set so `docker` can write to it. For instance, you might choose to put this folder at `~/losant-edge-agent` for easy access. For this example, we're going to create a folder at `/var/lib/losant-edge-agent/` and update the permissions to allow `docker` to write to it.
```

Comment:
Is filesystem a word? Should it be file system?

Reply:
Looks like either way? I'll update it since you are asking about it.

https://en.wikipedia.org/wiki/File_system

Reply:
the long debate of filesystem vs file system. I think we actually landed on file system last time. @cephirothdy2j do you remember?

Reply:
My vote is two words.

1. I like it more.
2. [Node's documentation](https://nodejs.org/api/fs.html) treats it as two words.

Reply:
Yeah, I grepped the docs and found 3 other uses where it's split.